frontend/components/Documents/Meeting.tsx

```diff
@@ -10,37 +11,53 @@ const MeetingPaper = styled(Paper)`
   flex-direction: column;
   padding: 1rem;
   margin-top: 1rem;
-  max-width: 40rem;
   overflow-wrap: anywhere;
 `;
 
-const File = styled(Box)`
-  margin-top: 1rem;
-`;
-
 export default function MeetingComponent({ meeting }: { meeting: Meeting }) {
   return (
     <MeetingPaper key={meeting.title}>
       <h2 style={{ margin: 0 }}>{meeting.title}</h2>
-      {meeting.files.map((file) => (
-        <File key={`file-${file.name}`}>
-          <Button
-            variant="contained"
-            target="_blank"
-            rel="noopener noreferrer"
-            href={file.thumbnailUrl}
-            download
-            fullWidth
-            sx={{
-              overflowWrap: 'anywhere',
-              justifyContent: 'flex-start',
-            }}
-          >
-            <ArticleIcon style={{ marginRight: '0.5rem' }} />
-            {file.name}
-          </Button>
-        </File>
-      ))}
+      <Box sx={{
+        display: 'grid',
+        gridTemplateColumns: 'repeat(auto-fill, minmax(250px, 1fr))',
+        gap: 2,
+      }}
+      >
+        {meeting.files.map((file) => {
+          const title = file.name.replaceAll('_', ' ');
+
+          return (
+            <Tooltip
```

Comment:
Tooltip doesn't fill much of a purpose?
![image](https://github.com/Dsek-LTH/member-page/assets/26229521/a7c79da6-20d2-4d99-be0e-b033953acbeb)

Reply:
For files with long file names they will be truncated (with ellipsis "..."). So the tooltip is there to see the full name.